index.html

```diff
@@ -5164,6 +5164,114 @@ <h2 id="AudioWorklet-concepts">
+                  false, throw a <code>TypeError</code> and abort all these
+                  steps.
+                </li>
+                <li>Let <code><em>prototype</em></code> be the result of <code>
+                  <a href=
+                  "http://www.ecma-international.org/ecma-262/6.0/#sec-get-o-p">
+                  Get</a>(O=<em>processorCtor</em>, P="prototype")</code>.
+                </li>
+                <li>If the result of <code><a href=
+                "http://www.ecma-international.org/ecma-262/6.0/#sec-ecmascript-data-types-and-values">
+                  Type</a>(argument=<em>prototype</em>)</code> is not
+                  <code>Object</code>, throw a <code>TypeError</code> and abort
+                  all these steps.
+                </li>
+                <li>If the result of <code><a href=
+                "http://www.ecma-international.org/ecma-262/6.0/#sec-iscallable">
+                  IsCallable</a>(argument=Get(O=<em>prototype</em>,
+                  P="process"))</code> is false, throw a <code>TypeError</code>
+                  and abort all these steps.
+                </li>
+                <li>If the result of <code><a href=
+                "http://www.ecma-international.org/ecma-262/6.0/#sec-get-o-p">
+                  Get</a>(O=<em>processorCtor</em>,
+                  P="parameterDescriptors")</code> is not an array or
+                  <code>undefined</code>, throw a <code>TypeError</code> and
+                  abort all these steps.
+                </li>
+                <li>Let definition be a new <a>AudioWorkletProcessor</a>
+                definition with:
+                  <ul>
+                    <li>node name being <em>name</em>
+                    </li>
+                    <li>processor class constructor being <em>constructor</em>
+                    </li>
+                  </ul>
+                </li>
+                <li>Add the key-value pair (<em>name</em> -
+                <em>constructor</em>) to the <a>node name to processor
+                definition map</a> of the associated
+                <a>AudioWorkletGlobalScope</a>.
+                </li>
+              </ol>
+              <p class="note">
+                The class constructor should only be looked up once, thus it
+                does not have the opportunity to dynamically change its
+                definition.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered. This key is used to look up the constructor of
+                  <a>AudioWorkletProcessor</a> during construction of an
+                  <a>AudioWorkletNode</a>.
+                </dd>
+                <dt>
+                  VoidFunction audioWorkletProcessorCtor
```

Comment:
Call this arg `processorCtor` to reconcile it with the above algorithmic description